mysql-test/suite/galera/t/galera_admin.test

```diff
@@ -82,6 +82,34 @@ SELECT COUNT(*) = 10 FROM x1;
 SELECT COUNT(*) = 10000 FROM t2;
 SELECT COUNT(*) = 10 FROM x2;
 
+
+# Check that super_read_only and read_only are handled properly (PXC-3923)
+--connection node_1
+--let super_read_only_saved = `SELECT @@super_read_only`
+--let read_only_saved = `SELECT @@read_only`
+
+# super_read_only
+SET GLOBAL super_read_only=1;
+--error ER_OPTION_PREVENTS_STATEMENT
+ANALYZE TABLE t1;
+
+# read_only
+SET GLOBAL super_read_only=0;
+SET GLOBAL read_only=1;
+
+CREATE USER 'testuser'@'%' IDENTIFIED WITH 'mysql_native_password' BY 'testuser';
+GRANT select, update, insert, delete ON *.* to 'testuser'@'%';
```

Comment:
Not a strong opinion though, could we have the SQL commands in uppercase?

Reply:
fixed